src/syscheckd/win_whodata.c

```diff
@@ -132,7 +132,7 @@ int set_winsacl(const char *dir, int position) {
             }
         break;
         case 1:
-            mdebug1(FIM_SCAL_NOCONFIGURE, dir);
+            // It is not necessary to configure the SACL of the directory
```

Comment:
If this message is no longer needed, please delete its definition https://github.com/wazuh/wazuh/blob/4980dd233544e49dd4bdae9c1f2b92f8e681699e/src/error_messages/debug_messages.h#L155

Reply:
Done 60483ac